src/resources/resources.controller.spec.ts

```diff
@@ -1,25 +1,271 @@
+                controller.createNewResource(requestMock, 9999, dtoCreateMock),
+            ).rejects.toThrow(NotFoundException);
+            expect(mockResourcesService.createNewResource).toHaveBeenCalledWith(
+                requestMock,
+                dtoCreateMock,
+                9999,
+            );
+        });
+    });
+    describe("findAllResources", () => {
+        it("findAllResources service should be defined", async () => {
+            expect(controller.findAllResources).toBeDefined();
+        });
+
+        it("should find all resources", async () => {
+            mockResourcesService.findAllResources.mockResolvedValueOnce(
+                mockResourceArr,
+            );
+
+            const result = await controller.findAllResources(
+                requestMock,
+                mockTeamId,
+            );
+
+            expect(result).toBeArray;
+            expect(result).toHaveLength(2);
+            expect(result[0]).toEqual({
+                id: expect.any(Number),
+                url: expect.any(String),
+                title: expect.any(String),
+                teamMemberId: expect.any(Number),
+                addedBy: {
+                    member: {
+                        firstName: expect.any(String),
+                        lastName: expect.any(String),
+                        id: expect.any(String),
+                        avatar: expect.any(String),
+                    },
+                },
+                createdAt: expect.any(Date),
+                updatedAt: expect.any(Date),
+            });
+            expect(mockResourcesService.findAllResources).toHaveBeenCalledWith(
+                requestMock,
+                mockTeamId,
+            );
+        });
+        it("should throw notFound exception for invalid teamId", async () => {
+            mockResourcesService.findAllResources.mockRejectedValueOnce(
+                new NotFoundException(),
+            );
+            expect(
+                controller.findAllResources(requestMock, 9999),
+            ).rejects.toThrow(NotFoundException);
+            expect(mockResourcesService.findAllResources).toHaveBeenCalledWith(
+                requestMock,
+                9999,
+            );
+        });
+    });
+    describe("Update Resource", () => {
+        it("updateResource service should be defined", async () => {
+            expect(controller.updateResource).toBeDefined();
+        });
+
+        it("should update a resource", async () => {
+            mockResourcesService.updateResource.mockResolvedValueOnce(
+                mockUpdatedResource,
+            );
+
+            const result = await controller.updateResource(
+                requestMock,
+                1,
+                dtoUpdateMock,
+            );
+
+            expect(result).toEqual(mockUpdatedResource);
+            expect(mockResourcesService.updateResource).toHaveBeenCalledWith(
+                requestMock,
+                1,
```

Comment:
try not to use magic numbers
```suggestion
                mockResource,id,
```